Exiled.Events/EventArgs/Player/SpawnedEventArgs.cs

```diff
@@ -27,9 +27,9 @@ public class SpawnedEventArgs : IPlayerEvent
         /// <param name="oldRole">the spawned player's old <see cref="PlayerRoleBase">role</see>.</param>
         /// <param name="roleChangeReason">the player's spawned with <see cref="RoleChangeReason" />.</param>
         /// <param name="roleSpawnFlags">the player's spawned with <see cref="RoleSpawnFlags" />.</param>
-        public SpawnedEventArgs(ReferenceHub player, PlayerRoleBase oldRole, RoleChangeReason roleChangeReason, RoleSpawnFlags roleSpawnFlags)
+        public SpawnedEventArgs(Player player, PlayerRoleBase oldRole, RoleChangeReason roleChangeReason, RoleSpawnFlags roleSpawnFlags)
         {
-            Player = Player.Get(player);
+            Player = Player.Get(player.ReferenceHub);
```

Comment:
Wtf you take Player.Referencehub inside a Player.Get() lol

Reply:
because the event parameters has been changed to `Player player` instead of `ReferenceHub player`
and if I try to take something that is not the referencehub it says that the player is null

Reply:
I created a null check but it says that the player is always null

Reply:
So I decided to make it like that, at least player is not null now

Reply:
oh right, I didn't see what do you mean